### ISSUE TITLE
TypeError: DCAE_HF.__init__() missing 1 required positional argument: 'model_name'

### ISSUE THREAD

**Issue by Aziily** [NVlabs/Sana#53]
When using the code in readme for inference without pre-downloading the models' weight, it seems that there will be such error.

I add one argument value in `diffusion/model/builder.py` to fix this.
To be specific, in line 88:
```python
# ========== Mark this row ==========
# dc_ae = DCAE_HF.from_pretrained(model_path).to(device).eval()
# ========== Replace it with ==========
dc_ae = DCAE_HF.from_pretrained(model_path, model_name=model_path.split("/")[-1]).to(device).eval()

```
This seems to be an error required for correction.

**Comment by lawrence-cj** [comment]
Sry, but we don't meet this problem. Maybe the `huggingface_hub=0.26.1` version is not the same with ours.
<img width="1266" alt="image" src="https://github.com/user-attachments/assets/1533bd02-2bc8-48a7-a090-e126d4bc2895">

**Comment by Aziily** [comment]
My huggingface version is 0.26.2, maybe this leads to an error?

**Comment by lawrence-cj** [comment]
If possible, could you try 0.26.1 to see if the bug will fix.

**Comment by Aziily** [comment]
I reinstall huggingface 0.26.1 and there seems to be no such error. Thank you for your help

**Comment by chenzebiaohub** [comment]
huggingface-hub          0.27.0 
==================
TypeError: DCAE_HF.__init__() missing 1 required positional argument: 'model_name'
dc_ae = DCAE_HF.from_pretrained(model_path).to(device).eval()
  0.26.1 also can't run

**Comment by Aziily** [comment]
> # huggingface-hub 0.27.0 @lawrence-cj
> TypeError: DCAE_HF.**init**() missing 1 required positional argument: 'model_name' dc_ae = DCAE_HF.from_pretrained(model_path).to(device).eval() 0.26.1 also can't run

If you use the replacement I mentioned above, will this bug be fixed? So maybe this is not an error about the package version but whether the checkpoint is downloaded?

**Comment by Aziily** [comment]
> Your meaning is that the model download weights might be incomplete or the path might be incorrect? @Aziily

Yeah, I mean maybe when the model downloaded, the model_name argument may not be required. So after downloading, the bug no longer exists.
But I don't know whether in your case the bug is fixed after the downloading, so this is just a guess.

**Comment by chenzebiaohub** [comment]
thanks ！I try again
